### DISCUSSION TITLE
FFUF: "append" wordlist

### DISCUSSION THREAD

**Discussion by TheTechromancer** [blacklanternsecurity/bbot#853] [Ideas]
It would be nice to be able to pass a second additional wordlist to the ffuf module.

@aconite33 @kerrymilan @liquidsec

**Comment by liquidsec** [comment]
Can you give an example use case? Where would be inserting values from the second list?

**Reply by TheTechromancer** [reply]
If you had a target-specific endpoint you wanted to try in addition to the usual stuff.

**Reply by liquidsec** [reply]
Gotcha, I thought you meant fuzzing two different spots. Now I understand. We can already swap the list with a custom one, is there a significant advantage to leaving the default list in when this is desired?

**Reply by liquidsec** [reply]
bump @TheTechromancer

**Reply by TheTechromancer** [reply]
I think this had been been requested by Mike / Kerry. Personally I think the current setup is fine.

**Reply by liquidsec** [reply]
@aconite33 any thoughts? Otherwise im gonna close this.

You can already add a custom wordlist currently.

**Comment by liquidsec** [comment]
we accidentally got this in this pr lol https://github.com/blacklanternsecurity/bbot/pull/3094
